src/main/java/org/gradlex/jvm/dependency/conflict/detection/rules/CapabilityDefinition.java

```diff
@@ -248,6 +248,10 @@ public enum CapabilityDefinition {
             "com.miglayout:miglayout-swing",
             "com.miglayout:miglayout"
     ),
+    MYSQL(HIGHEST_VERSION,
```

Comment:
This will used as the **name** of the **capability**. Which usually corresponds to one of the artifact names.

I think we should use

`MYSQL_CONNECTOR_JAVA` (the capability name is then `mysql-connector-j`)

Reply:
Hey @jjohannes, thanks for the review.

Yeah, makes sense actually, it would be easier to track/understand if gradle fails due to capability conflict (if using only conflict-detection).

Made the change and pushed it already.